### ISSUE TITLE
Extra Types of Actors roles

### ISSUE THREAD

**Issue by jellevanleeuwen** [opentripmodel/otm5-change-requests#20]
**Type of request**
- [ ] I'd like to add something to the OTM 5 spec
**Is your feature request related to a problem?**
If so, please describe the problem. For example: with the current specification I cannot model all of the actors that are involved in the transportation proces. 

**Describe the solution you'd like**
I would like to be free in the use of actor roles, the current roles are not sufficient enough, but i think they can stick as a standard, but i also want to be free to add roles, and use the same enumeration field. 

**Describe alternatives you've considered**
We are now using the same field, but this is not how you would (mis)use OTM.

**Additional context**

**Comment by bmeesters** [comment]
Which roles are you missing? I think it would not be a good idea to make the roles a free string. Since then some might use it inconsistently (using different names for the same thing,using different capitalization).

If you can be clear in which roles would be missing we can add those fairly easily and also ensure the standard is used consistently.

**Comment by jellevanleeuwen** [comment]
I am currently working for a client who have some extra roles, i will describe them briefly:

- Notify: actors that need to be notified of the events happening to the shipment
- Debtor/Invoice: actors that are invoiced for the shipment 
- Loader: actor that takes account of the loading of the shipment

Especially the loader and the Debtor are important to us.

**Comment by bmeesters** [comment]
I would say that debtor/invoice is the same as `consignee`. Since the consignee buys the goods, and therefore also gets the invoice. Or do you use the two differently? Otherwise I would suggest to just use that one.

loader/unloader are missing indeed. As are driver and subcontractor, since we are on the subject of missing roles. If I may ask, how do you use the loader actor? Do you use an actual person that is loading the goods, or is it a party that is providing someone to load the goods? And if so, is that a different party than the shipper/carrier?

Notify however is a bit different. Compared to the others it is not domain driven, but seems to be system driven. You cannot be a 'notify' whereas you can be any of the other roles. Therefore I don't think it would be a good role.

I will be sure to discuss them and see which ones we want to include.

**Comment by bmeesters** [comment]
We discussed the different roles and we see the following (that do no exist yet) as good to add:
* **driver**: the person operating the vehicle. Currently we do not support multiple drivers on a single trip. The work around would be to create multiple trips with different drivers.
*  **subcontractor**: the actual carrier of the data. However, from a contract point of view, the carrier accepting the assignment and the actually driving can be two different parties.

Debtor/Invoice should be part of `consignee` so I think that one is covered. Notifier doesn't really seem like a system role than a transport role, and for now we do not want to add it yet. If there are any arguments in favor we gladly hear them.

Loader/unloader we found difficult, because we do think it is part of actual transport (in contrast to notify), but we are also afraid that it will lead to a lot of other roles as well which seem rather minor. The recommendation is to use description instead for roles that are not there yet.
